docker/data-science/notebooks/ch12/0.setup.ipynb

```diff
@@ -305,11 +310,15 @@
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "aips.py  ch07  ch12  densevectors\t    session_gen.py\n",
-      "ch03\t ch08  ch13  ltr\t\t    solr_collection.py\n",
-      "ch04\t ch09  ch14  __pycache__\t    solr.py\n",
-      "ch05\t ch10  ch15  retrotech\t\t    webserver\n",
-      "ch06\t ch11  data  run_cell_extension.py  welcome.ipynb\n"
+      "aips.py\t\t    ch10\t\t    densevectors  retrotech\n",
```

Comment:
todo, check notebook and verify what this output is supposed to be

Reply:
ch12.0 directory issues have been corrected